commit 289e23aa03084b22c73ebdcf18371f1e6666ead0
Author: Andreas Arnez <arnez@linux.vnet.ibm.com>
Date:   Mon Sep 25 16:02:24 2017 +0200

    S390: Document guarded-storage register support
    
    This documents the newly added support for guarded-storage registers on
    IBM z.
    
    gdb/ChangeLog:
    
            * NEWS: Advertise support for guarded-storage registers on IBM z.
    
    gdb/doc/ChangeLog:
    
            * gdb.texinfo (S/390 and System z Features): Document the new
            features org.gnu.gdb.s390.gs and org.gnu.gdb.s390.gsbc.

### a/gdb/ChangeLog
### b/gdb/ChangeLog
## -1,5 +1,9 @@
 2017-09-25  Andreas Arnez  <arnez@linux.vnet.ibm.com>
 
+	* NEWS: Advertise support for guarded-storage registers on IBM z.
+
+2017-09-25  Andreas Arnez  <arnez@linux.vnet.ibm.com>
+
 	* s390-linux-nat.c (have_regset_gs): New static variable.
 	(s390_linux_fetch_inferior_registers): Handle guarded-storage
 	control block and guarded-storage broadcast control regsets.
Index: gdb-8.0.1/gdb/NEWS
===================================================================
--- gdb-8.0.1.orig/gdb/NEWS	2017-10-07 15:54:48.649165644 +0200
+++ gdb-8.0.1/gdb/NEWS	2017-10-07 15:55:01.006264129 +0200
@@ -3,6 +3,9 @@
 
 * Fortran: Support pointers to dynamic types.
 
+* GDB now supports access to the guarded-storage-control registers and the
+  software-based guarded-storage broadcast control registers on IBM z14.
+
 *** Changes in GDB 8.0
 
 * GDB now supports access to the PKU register on GNU/Linux. The register is
Index: gdb-8.0.1/gdb/doc/gdb.texinfo
===================================================================
--- gdb-8.0.1.orig/gdb/doc/gdb.texinfo	2017-10-07 15:54:44.891135692 +0200
+++ gdb-8.0.1/gdb/doc/gdb.texinfo	2017-10-07 15:54:48.660165731 +0200
@@ -41512,6 +41512,14 @@
 contain the 128-bit wide vector registers @samp{v16} through
 @samp{v31}.
 
+The @samp{org.gnu.gdb.s390.gs} feature is optional.  It should contain
+the 64-bit wide guarded-storage-control registers @samp{gsd},
+@samp{gssm}, and @samp{gsepla}.
+
+The @samp{org.gnu.gdb.s390.gsbc} feature is optional.  It should contain
+the 64-bit wide guarded-storage broadcast control registers
+@samp{bc_gsd}, @samp{bc_gssm}, and @samp{bc_gsepla}.
+
 @node Sparc Features
 @subsection Sparc Features
 @cindex target descriptions, sparc32 features
